keccak256/src/permutation/state_conversion.rs

```diff
@@ -24,14 +24,10 @@ impl<F: Field> StateBaseConversion<F> {
         flag: Column<Advice>,
     ) -> Self {
         meta.enable_equality(flag);
-        let bccs: [BaseConversionConfig<F>; 25] = state
-            .iter()
-            .map(|&lane| BaseConversionConfig::configure(meta, bi.clone(), lane, flag))
-            .collect::<Vec<_>>()
-            .try_into()
-            .unwrap();
+        let lane = meta.advice_column();
```

Comment:
We can replace the `state: [Column<Advice>; 25]` arg with `lane: Column<Advice>`, so we can avoid creating an advice_column here.

Reply:
Good catch!!!